localisation/000_wc_traits.csv

```diff
@@ -599,13 +599,13 @@ giant_desc;This character is quite oversized for its race. While imposing, this
 creature_giant;Giant;;Riese;;;;;;;;;;;x
 creature_giant_desc;This character is a giant.;;Dieser Charakter ist ein Riese.;;;;;;;;;;;x
 detached_priest;Dropout;;Aussteiger[From.GetinFemale];;;;;;;;;;;x
-detached_priest_desc;A dropout tried to get a good education, but didn't perfectly master any knowledge.;;Ein Aussteiger versuchte, eine gute Ausbildung zu erhalten, beherrschte aber noch kein vollst�ndiges Wissen.;;;;;;;;;;;x
+detached_priest_desc;This dropout tried to get a good education, but failed to master any discipline.;;Diese[From.GetrMale] Aussteiger[From.GetinFemale] bem�hte sich um eine gute Ausbildung, konnte aber keine Disziplin meistern.;;;;;;;;;;;x
 martial_cleric;Educated;;Gebildet;;;;;;;;;;;x
-martial_cleric_desc;This person got a decent education. [From.GetSheHeCap] doesn't need more.;;Diese Person hat eine anst�ndige Ausbildung erhalten. [From.GetSheHeCap] braucht nicht mehr.;;;;;;;;;;;x
+martial_cleric_desc;This person got a decent education. Surely, [From.GetSheHe] doesn't need any more.;;Diese Person hat eine anst�ndige Ausbildung erhalten. Sicherlich braucht [From.GetSheHe] nicht mehr.;;;;;;;;;;;x
 scholarly_theologian;Learned Person;;Gelehrte[From.GetrMale];;;;;;;;;;;x
 scholarly_theologian_desc;A learned person got a splendid education. [From.GetSheHeCap] is able to shine with [From.GetHerHis] mind.;;Eine gelehrte Person hat eine hervorragende Ausbildung erhalten. [From.GetSheHeCap] ist in der Lage, mit [From.GetHerHis]em Geist zu gl�nzen.;;;;;;;;;;;x
 mastermind_theologian;Outstanding Erudite;;Hervorragende[From.GetrMale] Erudit[From.GetinFemale];;;;;;;;;;;x
-mastermind_theologian_desc;This character possesses an outstanding mind with knowledge in many different areas of life.;;Dieser Charakter verf�gt �ber einen hervorragenden Verstand mit Kenntnissen in vielen verschiedenen Lebensbereichen.;;;;;;;;;;;x
+mastermind_theologian_desc;This character possesses an outstanding mind with knowledge and discipline in many different areas of life.;;Dieser Charakter verf�gt �ber einen hervorragenden Verstand mit Wissen und Erfahrung in vielen verschiedenen Lebensbereichen.;;;;;;;;;;;x
```

Comment:
Again, I'm not sure discipline works here. Isn't it kind of a general thing, like you are a disciplined person, not disciplined in the area of history? I might be wrong, but I've never heard it used that way. If you want something to go with knowledge, I'd suggest wisdom or insight.
```suggestion
mastermind_theologian_desc;This character possesses an outstanding mind with knowledge in many different areas of life.;;Dieser Charakter verf�gt �ber einen hervorragenden Verstand mit Wissen und Erfahrung in vielen verschiedenen Lebensbereichen.;;;;;;;;;;;x
```